whylog/assistant/regex_assistant/regex_match.py

```diff
@@ -46,8 +47,10 @@ def update_by_regex(self, new_regex):
 
         if new_regex[-1] != '$':
             new_regex += '$'
+        if new_regex[0] != '^':
```

Comment:
Because I assumed that for each parser in config we want to have a regex with "^". Is it right?